grandeur/download_files_ica.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+'''
+Author: Erin Young
+
+Description:
+
+This script will download the results from ica
+
+EXAMPLE:
+python3 download_files_ica.py -o <directory to download to> -r <run name>
+'''
+
+# trying to keep dependencies really low
+import argparse
+import subprocess
+import concurrent.futures
+import os
+
+version = '0.0.20230925'
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-o', '--out', type=str, help='directory where files will get downloaded to', default='ica')
+parser.add_argument('-r', '--run', type=str, help='sequencing run', required = True )
+parser.add_argument('-v', '--version', help='print version and exit', action='version', version='%(prog)s ' + version)
+args   = parser.parse_args()
+
+
```

Comment:
Before getting an id you will want to make sure that you are in the correct project. If not you are hoping that whoever used 'icav2' last was in the same project. For miseq runs the project you want is "Produciton", I know spelled wrong. The icav2 command is ```icav2 projects enter Produciton```.